src/main/java/uk/gov/justice/hmpps/casenotes/domain/audit/DeletedEntityListener.kt

```diff
@@ -41,6 +41,7 @@ class DeletedEntityListener {
   }
 
   private fun causeOfDelete() = when (getRequestPath()) {
+    null -> DeletionCause.MERGE
```

Comment:
Do you have any thoughts on how the proposed user entered `reason` value would get added to the deleted notes table?